docs/book/cookbook/application-wide-layout.md

```diff
@@ -0,0 +1,85 @@
+# Set Application-Wide Layout
+
+The format for the view helper `FlashMessenger` can be configured for an entire application and all uses of the helper.
+
+`Laminas\View\Helper\Service\FlashMessengerFactory` checks the application
+configuration and creates the view helper with the given format options.
+
+## Set Format for All Namespaces
+
+Add the following lines to the local or global configuration file, e.g. `config/autoload/global.config.php`:
+
+```php
+return [
+    'view_helper_config' => [
+        'flashmessenger' => [
+            'message_open_format'      => '<p%s>',
+            'message_close_string'     => '</p>',
+            'message_separator_string' => '<br>',
+        ],
+    ],
+];
+```
+
+## Set Formats Individually for Namespaces
+
+INFO: **New Feature**
+Available since version 1.8.0
+
+Add the following lines to the local or global configuration file, e.g. `config/autoload/global.config.php`:
+
+```php
+return [
+    'view_helper_config' => [
+        'flashmessenger' => [
+            'default' => [
+                'message_open_format'      => '<p%s>',
+                'message_close_string'     => '</p>',
+                'message_separator_string' => '<br>',
+                'classes'                  => 'custom-default example-class',
+            ],
+            'success' => [
+                'message_open_format'      => '<p%s>',
+                'message_close_string'     => '</p>',
+                'message_separator_string' => '<br>',
+                'classes'                  => 'custom-success example-class',
+            ],
+            'warning' => [
+                 // …
+            ],
+            'error'   => [
+                 // …
+            ],
+            'info'    => [
+                 // …
+            ],
+        ],
+    ],
+];
+```
+
+If the default unordered list should be retained, then set only the classes:
```

Comment:
This sentence does not tell me (reader) much. 

Is this just because the example above adds `<p%s>`-based rendering?

Reply:
> Is this just because the example above adds `<p%s>`-based rendering?

Right.